apps/preview/next/pages/showcases/FormFields/FormFieldsRequired.tsx

```diff
@@ -442,27 +439,25 @@ export default function FormFields() {
             {areaCodeIsInvalid && (
               <p className="mt-0.5 text-negative-700 typography-text-sm font-medium">No option selected</p>
             )}
-          </span>
-          <span className="mt-4 md:mt-0 grow">
-            <label htmlFor="phone-number" className="hidden">
-              Phone number
+          </label>
+          <span className="block mt-4 md:mt-0 grow">
+            <label>
+              <span className="typography-label-sm font-medium">Phone *</span>
+              <SfInput
+                value={phoneNumber}
+                type="tel"
+                invalid={phoneNumberIsInvalid}
+                required
+                placeholder="eg. 123 456 7890"
+                className="placeholder:text-neutral-500"
+                onInput={() => (phoneNumber ? setPhoneNumberIsInvalid(false) : setPhoneNumberIsInvalid(true))}
+                onBlur={() => (phoneNumber ? setPhoneNumberIsInvalid(false) : setPhoneNumberIsInvalid(true))}
```

Comment:
Btw. could you change this to:
```suggestion
                onChange={() => setPhoneNumberIsInvalid(!phoneNumber)}
                onBlur={() => setPhoneNumberIsInvalid(!phoneNumber)}
```